hydra/grammar/OverrideLexer.g4

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+// Regenerate lexer by running 'python setup.py antlr' at project root.
+// If you make changes here be sure to update the documentation (and update the grammar in command_line_syntax.md)
+lexer grammar OverrideLexer;
+
+
+// Re-usable Fragments.
+fragment DIGIT: [0-9];
+fragment INT_UNSIGNED: '0' | [1-9] (('_')? DIGIT)*;
+fragment CHAR: [a-zA-Z];
+fragment ID_: (CHAR|'_') (CHAR|DIGIT|'_')*;
+fragment INTERPOLATION_OPEN_: '${';
+fragment WS_: [ \t]+;
+fragment ESC_BACKSLASH_: '\\\\';  // escaped backslash
+
+/////////
+// KEY //
+/////////
+
+mode KEY;
+
+EQUAL: '=' WS_* -> mode(ARGS);
+
+TILDE: '~';
+PLUS: '+';
+AT: '@';
+COLON: ':';
+ATCOLON: '@:';
+SLASH: '/';
+
+ID: ID_;
+DOT_PATH: (ID | INT_UNSIGNED) ('.' (ID | INT_UNSIGNED))+;
+
+WS: WS_ -> skip;
```

Comment:
Using `-> skip` instead of `channel(hidden)` just because it seems more straightforward and achieves the same result (for what we want here) as far as I can tell.